youtrack_sdk/entities.py

```diff
@@ -327,3 +327,35 @@ class IssueLink(BaseModel):
     link_type: Optional[IssueLinkType] = Field(alias="linkType")
     issues: Optional[Sequence[Issue]]
     trimmed_issues: Optional[Sequence[Issue]] = Field(alias="trimmedIssues")
+
+
+class AgileInfo(BaseModel):
+    type: Literal["Agile"] = Field(alias="$type", default="Agile")
+    id: Optional[str]
+    name: Optional[str]
+
+
+class SprintInfo(BaseModel):
+    type: Literal["Sprint"] = Field(alias="$type", default="Sprint")
+    id: Optional[str]
+    name: Optional[str]
+    goal: Optional[str]
+    start: Optional[datetime]
+    finish: Optional[datetime]
+    archived: Optional[bool]
+    is_default: Optional[bool] = Field(alias="isDefault")
+    unresolvedIssuesCount: Optional[int]
+
+
+class Agile(AgileInfo):
+    owner: Optional[User]
+    visible_for: Optional[UserGroup] = Field(alias="visibleFor")
+    projects: Optional[Sequence[Project]]
```

Comment:
Or else maybe this should be moved to `AgileInfo`?

Reply:
Renamed to `AgileRef`.